packages/jsonresume-theme-flat/package.json

```diff
@@ -12,13 +12,7 @@
     "start": "resume serve"
   },
   "dependencies": {
-    "handlebars": "latest",
-    "prismjs": "^1.29.0"
+    "handlebars": "latest"
   },
-  "license": "MIT",
-  "devDependencies": {
-    "@babel/eslint-parser": "^7.24.5",
-    "eslint": "^9.3.0",
```

Comment:
Unfortunately this is quite dangerous. We use `eslint@8`. `eslint@9` brings breaking changes with regards to the [configuration](https://eslint.org/docs/latest/use/configure/migration-guide#custom-parsers) and bringing it into the project will create all kinds of nightmares.